netstandard/pkg/shims/netstandard/System.Security.Cryptography.Cng.Forwards.cs

```diff
@@ -2,6 +2,7 @@
 [assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(Microsoft.Win32.SafeHandles.SafeNCryptKeyHandle))]
 [assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(Microsoft.Win32.SafeHandles.SafeNCryptProviderHandle))]
 [assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(Microsoft.Win32.SafeHandles.SafeNCryptSecretHandle))]
+[assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Security.Cryptography.AesCng))]
```

Comment:
I'm surprised these three classes were in scope.  I thought of the Cng library as being "on" netstandard, not "in" netstandard, especially in 1.6.  But since they come from NetFx and are mainly about exposing persisted/named key support from the OS, I guess it could be a reason for a library to not be able to target netstandard.... ::shrug::.

LGTM, just sharing surprise.